### PR TITLE
Disable buttons, add mailto link, update copyright, correct spelling errors

### PR DESCRIPTION
**Pull request by newmano101**
Convert 'team@v1michigan.com' to a mailto link. Update copyright year. Correct spelling for 'StudyBuddies'. Change 'Design apps due' text to 'Apps now closed'. Disable and decrease opacity of 'Apply Now' and 'Let's do this' buttons.

### CODE REVIEW

File: components/Header.js [hunk 1]
```diff
@@ -30,12 +30,13 @@ const Header = () => (
         </div>
       </div>
       <div className="mt-6 text-center">
-        <a href="#apply" className="text-base md:text-xl font-bold outline-none text-white py-3 px-6 rounded-md bg-blue-700 hover:opacity-75">
+        <a  /*href="#apply"*/  className="text-base md:text-xl font-bold outline-none text-white py-3 px-6 rounded-md bg-blue-700 hover:opacity-75"
+        className="text-base md:text-xl m-2 font-bold outline-none text-white py-3 px-6 rounded-md bg-blue-700 /*hover:opacity-75*/ opacity-50 cursor-not-allowed">
```
Comment (elliotfklein) [inline components/Header.js]: `href` looks great!

Can we only have one `className` though? (The last one will overwrite the first one, but still, confusing to have two I think).

Also, looks like there are a few differences between the new and old `className` -- can we use what was there originally, just commenting out `hover:opacity-75` and adding `opacity-50 cursor-not-allowed`?

**Review by elliotfklein — CHANGES_REQUESTED** [review]
Looks good, thanks for making those changes everywhere, just a few minor comments.

File: components/Header.js (hunk 1 above)
Comment (newmano101) [reply components/Header.js]: Done.

**Review by elliotfklein — APPROVED** [review]
Perfect! :raised_hands:

**Pull request merged**